ntoskrnl/include/internal/cc.h

```diff
@@ -509,3 +509,7 @@ CcRosVacbDecRefCount(
 }
 #define CcRosVacbGetRefCount(vacb) InterlockedCompareExchange((PLONG)&(vacb)->ReferenceCount, 0, 0)
 #endif
+
+BOOLEAN
+CcRosFreeOneUnusedVacb (
```

Comment:
```suggestion
CcRosFreeOneUnusedVacb(
```

Reply:
In fact I wish we allowed this style, because it allows to distinguish header definitions from function invocations when grepping. This one is from Jerome's PR, btw.